doc/api-analytic-support.md

```diff
@@ -0,0 +1,442 @@
+100   692  100   692    0     0   2134      0 --:--:-- --:--:-- --:--:--  2217
+{
+   "apiVersion" : "v2",
+   "event" : {
+      ...
+      "readings" : [
+         {
+            "deviceName" : "Camera003",
+            "id" : "70545263-30e7-4c03-9741-0011300f2f9c",
+            "objectValue" : {
+               "SupportedAnalyticsModules" : {
+                  "AnalyticsModuleDescription" : [
+                     {
+                        "Fixed" : true,
+                        "MaxInstances" : 1,
+                        "Name" : "tt:Viproc",
+                        "Parameters" : {
+                           "SimpleItemDescription" : [
+                              {
+                                 "Name" : "Mode",
+                                 "Type" : "xs:string"
+                              },
+                              {
+                                 "Name" : "AnalysisType",
+                                 "Type" : "xs:string"
+                              }
+                           ]
+                        }
+                     }
+                  ]
+               }
+            },
+         }
+      ],
+      "sourceName" : "GetSupportedAnalyticsModules"
+   },
+   "statusCode" : 200
+}
+```
+```shell
+curl --request GET 'http://0.0.0.0:59882/api/v2/device/name/Camera003/GetAnalyticsModuleOptions?jsonObject=eyJDb25maWd1cmF0aW9uVG9rZW4iOiIxIn0=' | json_pp
+{
+   "apiVersion" : "v2",
+   "event" : {
+      "deviceName" : "Camera003",
+      "profileName" : "onvif-camera",
+      ...
+      "readings" : [
+         {
+            "deviceName" : "Camera003",
+            "id" : "43f0e59b-6f3e-4119-978e-299ccd59049d",
+            "objectValue" : {
+               "Options" : [
+                  {
+                     "AnalyticsModule" : "tt:Viproc",
+                     "Name" : "Mode",
+                     "StringItems" : {
+                        "Item" : [
+                           "Off",
+                           "Silent VCA",
+                           "Profile 1",
+                           "Profile 2",
+                           "Scheduled",
+                           "Event Triggered"
+                        ]
+                     }
+                  },
+                  {
+                     "AnalyticsModule" : "tt:Viproc",
+                     "Name" : "AnalysisType",
+                     "StringItems" : {
+                        "Item" : [
+                           "MOTION+",
+                           "Intelligent Video Analytics"
+                        ]
+                     }
+                  }
+               ]
+            },
+            ...
+            "resourceName" : "GetAnalyticsModuleOptions",
+            "valueType" : "Object"
+         }
+      ],
+      "sourceName" : "GetAnalyticsModuleOptions"
+   },
+   "statusCode" : 200
+}
+```
+**Note**: The jsonObject parameter is encoded from `{"ConfigurationToken": "{ANALYTIC_CONFIG_TOKEN}"}`
```

Comment:
For consideration in a later update, but perhaps add more details to convey where ANALYTIC_CONFIG_TOKEN comes from. Certainly it comes from the results of previous API's, but more text for clarity can be helpful. Same comment for similar usages later in the document.